Dockerfile

```diff
@@ -23,7 +23,7 @@ ENV NGINX_USER=nginx \
     NVM_VERSION=v0.39.1 \
     NODE_VERSION=16.14.0
 
-# Configure Python3.7 venv
+# Configure Python3.9 venv
```

Comment:
Maybe just remove the version here so it's all in one place? It's just not necessary to say what version in this particular comment.